### DISCUSSION TITLE
OMI glTF Working Group Meeting 2023-10-05

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/gltf-extensions#194] [General]
This meeting is on 2023-10-05 at 9:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Current Agenda: 

- Review KHR_audio

## Archive Agenda

* OMI Vehicles Extensions Research
* jin - proposals how to make our specs and work more accessible to others. showcase gltf models
* Review pull requests https://github.com/omigroup/gltf-extensions/pulls
* [OMI_link](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_link)
* [OMI_spawn_point](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_spawn_point)
* [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
  * [gltf-sample-assets](https://github.com/KhronosGroup/glTF-Sample-Models/pull/360)

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by antpb** [comment]
1. Physics: No changes by aaronfranke.
2. Audio: Extensions beyond the base spec
3.2.a Audio: Discussed how audio base spec only can use mp3 and wav. Ogg was suggested except for safari. The rest requires a codec implementation

**Comment by fire** [comment]
Brief notes:

1. Physics: No changes by aaronfranke.
2. Audio: Extensions beyond the base spec
3. Audio: Discussed how audio base spec only can use mp3 and wav. Ogg was suggested except for Safari's lack of support. All other audio codecs requires a codec implementation per platform.

**Comment by antpb** [comment]
anthony did this last week: https://www.youtube.com/watch?v=MsBhe39Oj7k
